frontend/src/app/Settings/Facility/FacilityFormContainer.tsx

```diff
@@ -174,12 +174,7 @@ const FacilityFormContainer: any = (props: Props) => {
   const appInsights = useAppInsightsContext();
   const [updateFacility] = useMutation(UPDATE_FACILITY_MUTATION);
   const [addFacility] = useMutation(ADD_FACILITY_MUTATION);
-  const trackSaveSettings = useTrackEvent(
-    appInsights,
-    "Save Settings",
-    null,
-    false
-  );
+  const trackSaveSettings = useTrackEvent(appInsights, "Save Settings", null);
```

Comment:
Pages wasn't working without that changes, as appInsight key not provided locally on my side that's why i'm also add this into this PR

Reply:
About hook, yes if we need only document title change, I can write a small hook, for that but, @TimBest do we will need any other head updates in the nearest future?

Reply:
We ended up making these `useTrackEvent` changes in a hotfix because it was causing production to break! If you merge `main` into this branch, these changes should go away.